<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDial</class>
 <widget class="QDialog" name="AccountDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Account</string>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UsernameInput</tabstop>
  <tabstop>PasswordInput</tabstop>
  <tabstop>DialogBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>DialogBtn</sender>
   <signal>rejected()</signal>
   <receiver>AccountDial</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtn</sender>
   <signal>accepted()</signal>
   <receiver>AccountDial</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
